test/fixtures/poro.rb

```diff
@@ -241,6 +252,22 @@ def self.root_name
   has_one :blog, key: :site
 end
 
+LinkTagSerializer = Class.new(ActiveModel::Serializer) do
+  has_many :posts, links: true
+end
+
+LinkNoDataTagSerializer = Class.new(ActiveModel::Serializer) do
+  has_many :posts, links: true, data: false
+end
+
+LinkUrlTagSerializer = Class.new(ActiveModel::Serializer) do
+  has_many :posts, links: { related: 'http://test.host/post_tags' }, data: false
+end
+
+LinkProcTagSerializer = Class.new(ActiveModel::Serializer) do
+  has_many :posts, links: { related: -> (object, name) { "http://test.host/tags/#{object.id}/#{name}/all" } }, data: false
```

Comment:
Maybe.. play with it and see if you like it.  It should be pretty simple to implement a simple subset ourselves to start with.